### PR TITLE
11081 Prevent GameRefresher changing visual order of pieces

### PR DESCRIPTION
**Pull request by BrentEaston**

### CODE REVIEW

**Comment by Cattlesquat** [comment]
I did a pretty big test of this with PoG and it seemed to work fine (and no longer changed the visual order of my pieces)

**Comment by riverwanderer** [comment]
I've tried build 8c048f1-RefresherUpdate on my dev version of C&C Napoleonics & on Montmirail & Vauchamps 1814. I refreshed a game and pre-defined  setups without incident. I will be working on C&C Nap again soon, which will be a more thorough proving out, but so far so good.

**Comment by BrentEaston** [comment]
Thanks guys. That's promising. I haven't looked at the other issue of the Decks in alternates Maps in pre-defined setups not being found for update yet.

**Comment by BrentEaston** [comment]
I believe this is ready to for 3.6.5. This fixes the change of visual order issues and was well tested by Mark and Brian.

The Decks on Alternate maps issue is a separate ticket that can wait for 3.6.6

File: vassal-app/src/main/java/VASSAL/build/module/GameRefresher.java [hunk 1]
```diff
@@ -394,12 +384,14 @@ else if (piece.getProperty(Mat.MAT_ID) != null) {
                   // If drawPile is owned by a specific board, then we can only match it if that board is active in this game
                   if (drawPile.getOwningBoardName() != null) {
                     if (map.getBoardByName(drawPile.getOwningBoardName()) == null) {
+                      System.out.println("A:" + map.getMapName() + " - " + drawPile.getOwningBoardName());
```
Comment (uckelman) [inline vassal-app/src/main/java/VASSAL/build/module/GameRefresher.java:387]: Diagnostic output should be removed?

File: vassal-app/src/main/java/VASSAL/build/module/GameRefresher.java [hunk 2]
```diff
@@ -394,12 +384,14 @@ else if (piece.getProperty(Mat.MAT_ID) != null) {
...
                       continue;
                     }
 
                     // If the drawPile is on a map that doesn't have its current owning board active, then we
                     // cannot match that drawPile.
                     if (drawPile.getMap().getBoardByName(drawPile.getOwningBoardName()) == null) {
+                      System.out.println("B:" + map.getMapName() + " - " + drawPile.getOwningBoardName());
```
Comment (uckelman) [inline vassal-app/src/main/java/VASSAL/build/module/GameRefresher.java:394]: Diagnostic output should be removed?

Comment (BrentEaston) [reply vassal-app/src/main/java/VASSAL/build/module/GameRefresher.java]: Fixe

File: vassal-app/src/main/java/VASSAL/build/module/GameRefresher.java (hunk 1 above)
Comment (BrentEaston) [reply vassal-app/src/main/java/VASSAL/build/module/GameRefresher.java]: fixed

**Pull request merged**